{
  "contest": 420,
  "handle": "FastChecker",
  "risk": "3",
  "title": "Various problems may arise due to incorrect implementation of `_mintInternal()`.",
  "issueId": 572,
  "issueUrl": "https://github.com/code-423n4/2024-07-traitforge-validation/issues/572"
}